docs/migrations.md

```diff
@@ -401,6 +401,32 @@ export class QuestionRefactoringTIMESTAMP implements MigrationInterface {
 }
 ```
 
+## Executing .sql files
+
+If your .sql file contains multiple statements per file, which is common with most standard sql tooling, you will first need to enable the `multipleStatements` option in the applicable `DataSource`.
```

Comment:
looks like `multipleStatements` is specific to some particular database types. How do we do for others?

Reply:
since this is for SQL files only, it should be applicable to all SQL based data sources, except for maybe SQLlite. 

I found it was required to use this option with PostgreSQL, even though the typeorm docs only show this option as available for MySQL/mariadb. 

As the option is to prevent SQL injection attacks, I would imagine it should be used with any SQL datasource

Reply:
right now based on the codebase `multipleStatements` flag is only available in `mysql` and `spanner` drivers. We need to add flag into typings as well for all other drivers which support this feature before we can add the docs. 

Because otherwise docs will confuse users who attempt to use `multipleStatements` in their driver and it result them a type error.